// console.log(a + b)
// console.log("This line will never be reached")

// you can manually throw errors by calling the error constructors and using the throw keyword 
throw new ReferenceError()
throw new SyntaxError()
throw new TypeError()